driver_cpl/bld/namelist_files/namelist_definition_drv.xml

```diff
@@ -2,7 +2,2782 @@
 
 <namelist_definition>
 
-  <version>1.0</version>
+<!-- Each namelist variable is defined in an <entry> element.  The
```

Comment:
This file is no longer in driver_cpl and should not be introduced as part of this PR. This file was part of the old namelist generation. These entries need to be merged with the new driver_cpl/cime_config/namelist_defaults_drv.xml which uses the new python nmlgen class and methods. The new namelist_defaults_drv.xml has both default values as well as definition values in one file.
Many of these settings should be the same in the new file.